forte/pipeline.py

```diff
@@ -120,6 +120,14 @@ def __init__(self, resource: Optional[Resources] = None):
             self.resource = resource
 
         self.initialized: bool = False
+        self._check_type_consistency: bool = False
+
+    def enforce_consistency(self):
```

Comment:
Not you can enforce the consistency but cannot disable it again. There are two options:
1) add a boolean as parameter here
2) use another function to disable it